crypto/engine/eng_devcrypto.c

```diff
@@ -391,9 +585,45 @@ static const struct digest_data_st *get_digest_data(int nid)
     return &digest_data[get_digest_data_index(nid)];
 }
 
+static void cryptodev_disable_all_digests(void)
+{
+    size_t i;
+
+    for (i = 0; i < OSSL_NELEM(digest_data); i++)
+        digest_data[i].enabled = 0;
+}
+
+static void cryptodev_enable_all_digests(void)
+{
+    size_t i;
+
+    for (i = 0; i < OSSL_NELEM(digest_data); i++)
+        digest_data[i].enabled = 1;
+}
+
+static int cryptodev_enable_digest_cb(const char *str, int len, void *unused)
+{
+    (void) unused;
+    char *name;
+    const EVP_MD *EVP;
+    size_t i;
+
+    if (len == 0 || (name = OPENSSL_strndup(str, len)) == NULL)
+        return 0;
+    EVP = EVP_get_digestbyname (name);
```

Comment:
Stile nit: remove the space between `EVP_get_digestbyname` and the parenthesis

Reply:
fixed it.